common/v2/components/Checkbox.scss

```diff
@@ -0,0 +1,87 @@
+$border-color: #e8eaed;
+$bright-sky-blue: #1eb8e7;
+$checkbox-size: 20px;
+$border-radius: 2px;
+
+/*
+  Styling Checkbox Starts
+  Adapted from: https://appitventures.com/blog/styling-checkbox-css-tips/
+*/
+.checkbox-label {
+  display: block;
+  position: relative;
+  cursor: pointer;
+  font-size: 18px;
+  line-height: $checkbox-size;
+  height: $checkbox-size;
+  width: $checkbox-size;
+  clear: both;
+  margin: 0;
+}
+
+.checkbox-label input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+}
+
+.checkbox-label .checkbox-custom {
+  position: absolute;
+  height: $checkbox-size;
+  width: $checkbox-size;
+  background-color: transparent;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+  transition: all 0.2s ease-out;
+}
+
+.checkbox-label input:checked ~ .checkbox-custom {
+  background-color: transparent;
+  border-radius: $border-radius;
+  transform: rotate(0deg) scale(1);
+  opacity: 1;
+  border: 1px solid $border-color;
+}
+
+.checkbox-label input:checked ~ .checkbox-custom {
+  background-color: transparent;
+  border-radius: $border-radius;
+  transform: rotate(0deg) scale(1);
+  opacity:1;
+  border: 1px solid $border-color;
+}
+
+// Create the pseudo content element and place it in the center
+// Hide it by setting the scale(0)
+// This will be the origin of the transition.
+.checkbox-label .checkbox-custom::after {
+  position: absolute;
+  content: "";
+  left: 50%;
+  top: 50%;
+  margin-top: -1px;
+  margin-left: -1px;
+  height: 0px;
+  width: 0px;
+  border: 1px solid $bright-sky-blue;
+  opacity:1;
+  transform: rotate(0deg) scale(0);
+  transition: all 0.2s ease-out;
+}
+
+// When the checkbox is selected we transform to show half the borders
+// of a rotated rectangle which provides the swoosh effect.
+// 'top' & 'left' are used to position the swoosh with the container.
+// 'widht' & 'height' serve as the size.
+.checkbox-label input:checked ~ .checkbox-custom::after {
+  opacity:1;
+  left: 35%;
+  top: 15%;
+  width: 7px;
+  height: 12px;
+  border: solid $bright-sky-blue;
+  border-radius: 1px;
+  border-width: 0 2px 2px 0;
+  background-color: transparent;
+  transform: rotate(45deg) scale(1);
+}
```

Comment:
I'd rather stay consistent with whether a specific feature uses SCSS or SC styles (with the exception of specificity overrides). It seems like everything in Checkbox.scss could be moved to styled components which I'd prefer, though you could do the opposite if you're more comfortable with SCSS.

Reply:
I'll push these changes tomorrow.

Reply:
I moved all the Checkbox and AccountDropdown to SC and created /v2/theme that combines @mycrypto/ui theme with GAU theme so it's accessible with theThemeProvider